packages/account/src/Components/Routes/route-with-sub-routes.jsx

```diff
@@ -39,6 +39,17 @@ const RouteWithSubRoutes = route => {
 
         const title = route.getTitle?.() || '';
         document.title = `${title} | ${default_title}`;
+
+        const current_search_params = new URLSearchParams(location.search);
+        const current_lang = current_search_params.get('lang') ? `${current_search_params.get('lang').toLowerCase()}` : 'en';
+
+        const alternate_rel_link = document.querySelector("link[rel='alternate']")
+        const canonical_rel_link = document.querySelector("link[rel='canonical']")
+
+        alternate_rel_link.setAttribute('hreflang', current_lang);
+        alternate_rel_link.setAttribute('href', `${location.origin}${location.pathname}`);
+        canonical_rel_link.setAttribute('href', `${location.origin}${location.pathname}`);
```

Comment:
The code is being duplicated 5 times.
Define it once as a function and reuse it.